pkg/cloudprovider/provider/gce/config.go

```diff
@@ -109,6 +110,7 @@ type config struct {
 	multizone             bool
 	regional              bool
 	customImage           string
+	podNetworkFamily      util.NetworkFamily
```

Comment:
I would recommend to name this `networkFamily` instead of `podNetworkFamily`, as this is actually the network family of the node.

Reply:
ah yes.